modules/text2vec-openai/clients/vectorizer.go

```diff
@@ -54,7 +54,12 @@ type openAIApiError struct {
 
 func buildUrl(baseURL, resourceName, deploymentID string, isAzure bool) (string, error) {
 	if isAzure {
-		host := "https://" + resourceName + ".openai.azure.com"
+		host := baseURL
+		if host == "" {
```

Comment:
this actually won't work, bc default `baseURL` is `https://api.openai.com` One quick fix for that would be to change this condition to:
```
if host == "" || host == "https://api.openai.com" {
```

Reply:
Thanks. I updated it to use the constant defined in the vectorizer module. Using an existing constant seems preferable but it looks like that introduces a dependency of the client package on the vectorizer subpackage. That didn't seem to cause a circular dependency but maybe there's more context I'm missing.

Reply:
yeah, I noticed that, hmm, I'm ok with just `host == "https://api.openai.com"` without the `vectorizer2` import.
One question, do you want this change to be rolled out quickly? bc you have targeted this change for `master` if you want this to be released with our next patch release then please base your PR changes on `stable/v1.23` branch that way it will be released a lot sooner

Reply:
Done. Sooner would be great. I rebased on `stable/v1.23`.
PTAL

Reply:
looks good